week4/lc567_checkInclusion.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @param {string} s1
+ * @param {string} s2
+ * @return {boolean}
+ */
+var checkInclusion = function (s1, s2) {
+    if (s1.length > s2.length) {
+        return false;
+    }
+
+    let s1Arr = new Array(26).fill(0);
+    for (let i = 0; i < s1.length; i++) {
+        s1Arr[s1.charCodeAt(i) - "a".charCodeAt(0)]++;
+    }
+
+    let left = 0,
+    counter = 0;
+
+    for (let right = 0; right < s2.length; right++) {
+        if (s1Arr[s2.charCodeAt(right) - "a".charCodeAt(0)] >= 1) {
+            counter++;
+        }
```

Comment:
Just having one small problem, I think we should indent the code from line23 to line34.